views/update/overseas-entity-query.html

```diff
@@ -12,15 +12,30 @@
 {% endblock %}
 
 {% block content %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
         <h1 class="govuk-heading-xl">{{title}}</h1>
+        {% if errorList %}
```

Comment:
Why creating the error list when you have already the validation middleware attached to it? It should do the work for you by throwing the correct error message

Reply:
We would be loading the entity from the validator. We've chosen to do that in the controller but with a validator that checks the format.

Reply:
Why spreading the validation logic?, could you please follow the common pattern

Reply:
@mouhajer-ch removed